src/System.Net.Sockets/tests/FunctionalTests/AcceptAsync.cs

```diff
@@ -8,7 +11,7 @@ namespace System.Net.Sockets.Tests
 {
     public class AcceptAsync
     {
-        private const int TestPortBase = 8000;
+        private const int TestPortBase = 7000;
```

Comment:
Some tests will always use allocated ports. We could envision a retry mechanism but never completely remove all tests from allocating their own ports.